test/minErrSpec.js

```diff
@@ -68,6 +73,27 @@ describe('minErr', function() {
     expect(myError.message).toMatch(/a is {"b":{"a":"..."}}/);
   });
 
+  it('should handle arguments that are objects with max depth', function() {
+    var a = {b: {c: {d: 1}}};
+
+    configureErrorHandling({objectMaxDepth: 1});
+    var myError = testError('26', 'a when objectMaxDepth=1 is {0}', a);
+    expect(myError.message).toMatch(/a when objectMaxDepth=1 is {"b":"..."}/);
+
+    configureErrorHandling({objectMaxDepth: 2});
+    myError = testError('26', 'a when objectMaxDepth=2 is {0}', a);
+    expect(myError.message).toMatch(/a when objectMaxDepth=2 is {"b":{"c":"..."}}/);
+  });
+
+  they('should handle arguments that are objects and ignore max depth when objectMaxDepth = $prop',
+      [NaN, null, undefined, true, false, -1, 0], function(maxDepth) {
+        var a = {b: {c: {d: 1}}};
+
+        configureErrorHandling({objectMaxDepth: maxDepth});
+        var myError = testError('26', 'a is {0}', a);
+        expect(myError.message).toMatch(/a is {"b":{"c":{"d":1}}}/);
+  });
```

Comment:
I have added this lines

      configureErrorHandling({objectMaxDepth: originalObjectMaxDepthInErrorMessage});
      myError = testError('26', 'a when objectMaxDepth is default=5 is {0}', a);
      expect(myError.message).toMatch(/a when objectMaxDepth is default=5 is {"b":{"c":{"d":     {"e":{"f":"..."}}}}}/);

to the previous case.